### PR TITLE
c4 review

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Fixed now on the PR, sorry for creating a storm in a teacup.

I guess it was a temporary issue associated with one of these commits but hard to decipher without looking at the changes from the messages!

image

**Comment by Nowosad** [comment]
@Robinlovelace ready for review?

**Comment by Robinlovelace** [comment]
Ready for review, finally! Thanks for the nudges @Nowosad !

**Comment by Robinlovelace** [comment]
Nudge on this...

**Comment by Nowosad** [comment]
Hi @Robinlovelace - I remember about this. I plan do my review on Thursday.

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -598,17 +712,21 @@ The next subsection explores these and related operations in more detail.
 ### Map algebra
 
 \index{map algebra}
-Map algebra makes raster processing really fast.
-This is because raster datasets only implicitly store coordinates.
-To derive the coordinate of a specific cell, we have to calculate it using its matrix position and the raster resolution and origin.
+The term 'map algebra' was coined in the late 1970s to describe a "set of conventions, capabilities, and techniques" for the analysis of geographic raster *and* (although less prominently) vector data [@tomlin_map_1994].
+Although the concept never became widely adopted, the term usefully encapsulates and helps classify the range operations that can be undertaken on raster datasets.
```
[the remark this answers is not in the document]
Comment (Nowosad) [reply 04-spatial-operations.Rmd:716]: If you meant "vector" map algebra -- I think we can drop this piece of history from this section.

File: geocompr.bib [hunk 2]
```diff
@@ -531,14 +531,6 @@ @article{douglas_algorithms_1973
   keywords = {\#nosource}
 }
 
-@manual{dunnington_ggspatial_2021,
```
Comment (Nowosad) [inline geocompr.bib:534]: @Robinlovelace this .bib removes some of the new references...

**Comment by Robinlovelace** [comment]
> @Robinlovelace this .bib removes some of the new references...

That's not a problem, we can sort out bibliographies post merge.

**Review by Nowosad — APPROVED** [review]

**Pull request merged**